app/src/main/res/layout/activity_live_matches.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:orientation="vertical" android:paddingLeft="10dp" android:paddingRight="10dp">
+    <ListView
```

Comment:
DONT USE LISTVIEW. Use RecyclerView. 99% of the time, RecyclerView is more optimal, especially in this case where you're reusing the same layout anyway for each livematch.

Reply:
So in the the discussion [here](https://stackoverflow.com/questions/28392554/should-we-use-recyclerview-to-replace-listview) they say "If ListView works for you, there is no reason to migrate." I guess it sucks a bit that I wasn't referred to RecyclerView directly when searching for how to create a simple list, but I don't really see a good reason to put more effort into this again to migrate to RecyclerView. Especially since I'm planning to address UI devs in the Discord channel later to give them a chance to gain more experience rewriting parts of this app's UI.

Reply:
Decided to migrate to RecyclerView despite my original comment on this